versions/3.1.1.md

```diff
@@ -2698,14 +2693,23 @@ components:
 
 #### <a name="discriminatorObject"></a>Discriminator Object
 
-When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object can be used to aid in serialization, deserialization, and validation.  The discriminator is a specific object in a schema which is used to inform the consumer of the document of an alternative schema based on the value associated with it.
+When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object can be used to aid in serialization, deserialization, and validation. The `discriminator` keyword is used to inform the consumer of the document which of the alternatives is expected or preferred.
```

Comment:
I find the phrase "expected or preferred" strange.  Also, I'm not sure how "consumer" is defined in this sentence but it can be used by both the consumer and server.

```suggestion
When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object gives a hit about the expected schema of the document. It can be used to aid in serialization, deserialization, and validation.
```
Also, why interchange "keyword" and "object"?

Reply:
> why interchange "keyword" and "object"?

"Keyword" refers to the key and "object" refers to the value. The `discriminator` keyword has an object as its value.